### PR TITLE
Verific parser

### PR DESCRIPTION
**Pull request by alaindargelas**

### CODE REVIEW

**Comment by alaindargelas** [comment]
@akashlevy this is ready to review

File: liberty2json.cpp [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
 	// Setup signal handling
 	backward::SignalHandling sh;
-
 	// Parse arguments
   argparse::ArgumentParser program("liberty2json");
   program.add_argument("filename").help("name of the Liberty file to parse");
 	program.add_argument("--outfile").help("name of the output JSON file");
 	program.add_argument("--check").help("check the Liberty file for errors").flag();
 	program.add_argument("--debug").help("enable debug mode").flag();
+	program.add_argument("--indent").help("enable debug mode").flag();
```
Comment (akashlevy) [inline liberty2json.cpp:17]: Help comment is not right

File: liberty2json.cpp [hunk 2]
```diff
@@ -1,20 +1,22 @@
...
 	program.add_argument("--ignore-complex-attrs").help("ignore complex attributes").flag();
+	program.add_argument("--verific").help("Verific parser").flag();
```
Comment (akashlevy) [inline liberty2json.cpp:19]: "Use Verific parser"

File: liberty2json.cpp [hunk 3]
```diff
@@ -23,18 +25,19 @@ int main(int argc, char *argv[]) {
     std::cerr << program;
     return 1;
   }
-
+
```
Comment (akashlevy) [inline liberty2json.cpp:28]: Remove this whitespace

**Review by akashlevy — CHANGES_REQUESTED** [review]

**Pull request closed without merging**